index.html

```diff
@@ -132,7 +142,7 @@ <h2>Sponsors</h2>
       </section>
 
 
-      <section>
+      <section id="footer">
         <div>
           <h3>Support</h3>
           <ul>
```

Comment:
I like how you utilize different fonts from Google Font to make your page looks unique. You followed the project requirement closely and placed the normalize.css style file before your style file. That's great! For the Google Font, you could also add it to your css file using @import url (this would not require adding this font to the html file). :)